src/Equinox.EventStore/EventStore.fs

```diff
@@ -635,12 +697,12 @@ type Connector
         .LimitRetriesForOperationTo(reqRetries) // ES default: 10
         |> fun s ->
             match node with
-            | NodePreference.Master -> s.PerformOnMasterOnly()  // explicitly use ES default of requiring master, use default Node preference of Master
-            | NodePreference.PreferMaster -> s.PerformOnAnyNode() // override default [implied] PerformOnMasterOnly(), use default Node preference of Master
+            | NodePreference.Master         -> s.PerformOnMasterOnly()                  // explicitly use ES default of requiring master, use default Node preference of Master
```

Comment:
Nice CC @fnipo see other people like Excel formatting

Reply:
it depends :p